src/WebJobs.Script.WebHost/FileMonitoringService.cs

```diff
@@ -275,7 +217,7 @@ public void Dispose()
         internal static async Task SetAppOfflineState(string rootPath, bool offline)
         {
             string path = Path.Combine(rootPath, ScriptConstants.AppOfflineFileName);
-            bool offlineFileExists = File.Exists(path);
+            bool offlineFileExists = System.IO.File.Exists(path);
```

Comment:
Ahh! Good catch. I had it before because I had added a reference to `using Microsoft.Azure.WebJobs.Script.Eventing` which also had a definition for `File` and it was causing a collision with `System.IO.File`. 

But I removed that dependency and forgot to revert this. Thanks, will update!